leaf-ui/rappid-extensions/ActorInspector.js

```diff
@@ -4,19 +4,16 @@ var ENTER_KEY = 13;
 var ActorInspector = Backbone.View.extend({
         model: joint.shapes.basic.Actor,
         
-        initialize: function() {
```

Comment:
Bring back initialize and use it to set this.actor = this.model.get('actor') so that you can access it easily throughout the file